<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Register</class>
 <widget class="QWidget" name="Register">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>244</width>
    <height>322</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>37</width>
         <height>17</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="minimizeButton">
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">#minimizeButton {
    border: none;
    background-color: transparent;
    min-width: 30px;
    min-height: 30px;
    max-width: 30px;
    max-height: 30px;
}

#minimizeButton:hover {
    background-color: #E1E1E1;
}

#minimizeButton:pressed {
    background-color: #BDBDBD;
}</string>
       </property>
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="closeButton">
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">#closeButton {
    border: none;
    background-color: transparent;
    min-width: 30px;
    min-height: 30px;
    max-width: 30px;
    max-height: 30px;
}

#closeButton:hover {
    background-color: #E57373;
    color: white;
}

#closeButton:pressed {
    background-color: #D32F2F;
    color: white;
}</string>
       </property>
       <property name="text">
        <string>×</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_title">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>50</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>11</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 11pt &quot;微软雅黑&quot;;</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:20pt;&quot;&gt;注册&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>17</width>
       <height>16</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEdit_username">
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>-1</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {
    background-color: #f0f0f0;
    border: 1px solid #ccc;
    border-radius: 5px;
    padding: 5px;
    color: #333;
	font-family: Arial, sans-serif;
    font-size: 14px;
    font-weight: normal;
}

QLineEdit:hover {
    border-color: #aaa;
}

QLineEdit:focus {
    border-color: #0078d7;
    background-color: #fff;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>用户名</string>
     </property>
     <property name="clearButtonEnabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEdit_pwd">
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>-1</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {
    background-color: #f0f0f0;
    border: 1px solid #ccc;
    border-radius: 5px;
    padding: 5px;
    color: #333;
	font-family: Arial, sans-serif;
    font-size: 14px;
    font-weight: normal;
}

QLineEdit:hover {
    border-color: #aaa;
}

QLineEdit:focus {
    border-color: #0078d7;
    background-color: #fff;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>密码</string>
     </property>
     <property name="clearButtonEnabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEdit_pwd_again">
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>-1</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {
    background-color: #f0f0f0;
    border: 1px solid #ccc;
    border-radius: 5px;
    padding: 5px;
    color: #333;
	font-family: Arial, sans-serif;
    font-size: 14px;
    font-weight: normal;
}

QLineEdit:hover {
    border-color: #aaa;
}

QLineEdit:focus {
    border-color: #0078d7;
    background-color: #fff;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>再次输入密码</string>
     </property>
     <property name="clearButtonEnabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>14</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>37</width>
         <height>17</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="button_signUp_config">
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>30</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>11</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	font: 11pt &quot;微软雅黑&quot;;
    background-color:rgb(25,198,136); 
    color: rgb(255,255,255);          
    border-radius: 15px;
}              
QPushButton:hover{
    background-color:rgb(20,168,106);
    color: rgb(255,255,255); 
    border-radius: 15px;
}
QPushButton:pressed{
    background-color:rgb(15,138,76);
    color: rgb(255,255,255); 
    border-radius: 15px;
}
QPushButton:disabled{
    background-color:rgb(143,217,152);
    color: rgb(255,255,255); 
    border-radius: 15px;
}</string>
       </property>
       <property name="text">
        <string>确认</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="LabelWithClick" name="label_return_login">
     <property name="styleSheet">
      <string notr="true">QLabel{color:rgb(25,198,136)} 
QLabel:hover{color:rgb(20,168,106)}          
QLabel:pressed{color:rgb(15,138,86)}    </string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;返回登录&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LabelWithClick</class>
   <extends>QLabel</extends>
   <header location="global">labelwithclick.h</header>
   <slots>
    <signal>clicked()</signal>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
